android/library/src/main/java/com/liferay/mobile/screens/util/ServiceVersionFactory62.java

```diff
@@ -77,4 +77,14 @@ public JournalContentConnector getJournalContentConnector(Session session) {
 	public ScreensJournalContentConnector getScreensJournalContentConnector(Session session) {
 		return new ScreensJournalContentConnector62(session);
 	}
+
+	@Override
```

Comment:
No, but it's recommended: http://stackoverflow.com/questions/212614/should-we-override-an-interfaces-method-implementation